README.md

```diff
@@ -94,3 +80,40 @@ graph LR
   click ioserver "https://github.com/zowe/zowe-native-proto/blob/main/doc/server/ioserver_architecture.md"
   click zowex "https://github.com/zowe/zowe-native-proto/blob/main/doc/server/zowex_architecture.md"
 ```
+
+## Troubleshooting
+
+### go: FSUM7351 not found
+
+Ensure go is part of PATH
+
+### Client connection error - Error: All configured authentication methods failed
+
+Check that your username and password are correct.<br/>
+For private keys, confirm that you can ssh into the LPAR/zVDT using it.
+
+### FSUM9383 Configuration file `/etc/startup.mk' not found
+
+You should be able to find the startup.mk file in `/samples`
+
+- `cp /samples/startup.mk /etc/startup.mk` <br/>
+  _source:_ https://www.ibm.com/support/pages/fsum9383-configuration-file-etcstartupmk-not-found
+
+### Building zut.o - FSUM3221 xlc++: Cannot spawn program /usr/lpp/cbclib/xlc/exe/ccndrvr
+
+The only way we (DKelosky) knew how to fix this is via SYSVIEW commands.
```

Comment:
we can replace this line with something like: 

> One workaround is to add CBC.SCCNCMP" to your system LINKLIST concatenation.  Here is an example doing this via SYSVIEW commands

An untested alternative would be to add `CBC.SCCNCMP` to [steplib](https://www.ibm.com/docs/en/zos/3.1.0?topic=shell-building-steplib-environment-steplib-environment-variable).  Then, we won't need SYSVIEW specific commands and we'll have a safer alternative to modifying LINKLIST.